### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by ealfonsov89** [alexandrim0/p2plib-dart#3] [Q&A] [answered]
Hi folks

I'm interested on this package to create a p2p between two instances of the same app on flutter on different operating systems. I can not find the documentation, if I click on the link, it sends me to a dead page.

Please, help me to found documentation.

Regards

**Comment by alexandrim0** [comment, marked as answer]
Hi, @ealfonsov89 
Sorry, I missed this.

Is it still actual for you?
You can ask me what you need or use source code of application uses this lib - https://github.com/alexandrim0/keyper 

I hope, this package will be documented better at some moment.

**Reply by ealfonsov89** [reply]
Hi, thanks, but I manage to fulfill my requirements with a secure websocket implementation of my own. I know what it is like to work by yourself along in a project, so I get you

**Reply by danbachar** [reply]
Hi! It would be best if we could write together a documentation for bootstrapping a connection to a server as a client device: maybe using one of the publicly available servers [here?](https://gist.github.com/mondain/b0ec1cf5f60ae726202e)

**Reply by alexandrim0** [reply]
> Hi! It would be best if we could write together a documentation for bootstrapping a connection to a server as a client device: maybe using one of the publicly available servers [here?](https://gist.github.com/mondain/b0ec1cf5f60ae726202e)

Hi!
First of all, thank you!
This lib does not use any STUN.
This lib implements simple interface and protocol, by design.
And I do prefer to keep it simple.
It only allows peer-to-peer connection with local devices and between real IPs.
In other case it uses relay.

**Reply by alexandrim0** [reply]
Also I belive, it is possible to write another package based on this  lib to do NAT traversal.
